File: Domain/Education/StudiskaPrograma.cs
using System;

namespace LearnByPractice.Domain.Education
{
    /// <summary>Класа за студиска програма.</summary>
    public class StudiskaPrograma
    {
        /// <summary>Шифра </summary>
        /// <value>Ја зема/поставува шифрата на студиската програма</value>
        public int Id { get; set; }

        /// <summary>Име</summary>
        /// <value>Го зема/поставува името на студиската програма</value>
        public String Ime { get; set; }

        /// <summary> Конструктор на класата <c>StudiskaPrograma</c>, без параметри. </summary>
        public StudiskaPrograma() { }
    }
}

File: UI.Web.SPA/wwwroot/app/model/educataion/studiskaprograma.js
var app;
(function (app) {
    var model;
    (function (model) {
        "use strict";
        var education;
        (function (education) {
            var StudiskaPrograma = (function () {
                function StudiskaPrograma() {
                }
                return StudiskaPrograma;
            }());
            education.StudiskaPrograma = StudiskaPrograma;
        })(education = model.education || (model.education = {}));
    })(model = app.model || (app.model = {}));
})(app || (app = {}));
